packages/registry/README.md

```diff
@@ -57,13 +57,13 @@ Fill in the steps for filing a pull request https://github.com/immersive-web/web
 ![Diagram of top-level schema parts](./figures/Concepts.png)
 
 ### Profile id and fallback profile ids
-Profiles are required to have a `profileId` that uniquely identifies the profile and must conform to the format specified by the [WebXR Device API](http://www.w3.org/tr/webxr). Profiles are also required to have a `fallbackProfileIds` array containing profile ids that are considered acceptable substitutes for profile. This array may be empty, but, if it is not, the ids it contains must match another existing profile detailed in this repository. 
+Profiles are required to have a `profileId` that uniquely identifies the profile and must conform to the format specified by the [WebXR Device API](http://www.w3.org/tr/webxr). A `profileId` cannot be changed once added to the registry; if changes are necessary a new `profileId` must be defined. Profiles are also required to have a `fallbackProfileIds` array containing profile ids defined in the registry that are considered acceptable fallback substitutes for profile. Profiles with the `generic` prefix may have this array empty.  All other profiles must have at least one fallback profile id, and the the last entry in the array must have the `generic` vendor prefix. 
```

Comment:
To properly address #99 can we also mention the stability rules? I.e. "the contents of the fallback list may change, though in the most part changes will likely be in the form of additions"